app/components/review_draft/draft/view.html.erb

```diff
@@ -27,6 +27,8 @@
         component.row(**row_helper(@trainee, :course_details))
       end
 
+      component.row(**row_helper(@trainee, :placement_details))
```

Comment:
```suggestion
if require_feature_flag(:trainee_placement)
              component.row(**row_helper(@trainee, :placement_details))
            end

```

May need something like that to hide or show thing...

Reply:
done